doc/supported-platforms.md

```diff
@@ -2,11 +2,11 @@
 
 Ignition is currently only supported for the following platforms:
 
-* [Bare Metal] - Use the `coreos.config.url` kernel parameter to provide a URL to the configuration. The URL can use the `http://`, `https://`, `tftp://`, or `s3://` schemes to specify a remote config or the `oem://` scheme to specify a local config, rooted in `/usr/share/oem`.
-* [PXE] - Use the `coreos.config.url` and `coreos.first_boot=1` (**in case of the very first PXE boot only**) kernel parameters to provide a URL to the configuration. The URL can use the `http://`, `https://`, `tftp://`, or `s3://` schemes to specify a remote config or the `oem://` scheme to specify a local config, rooted in `/usr/share/oem`.
+* [Bare Metal] - Use the `ignition.config.url` kernel parameter to provide a URL to the configuration (also `flatcar.config.url` OR `coreos.config.url` are accepted). The URL can use the `http://`, `https://`, `tftp://`, or `s3://` schemes to specify a remote config or the `oem://` scheme to specify a local config, rooted in `/usr/share/oem`.
+* [PXE] - Use the `ignition.config.url` and `coreos.first_boot=1` (**in case of the very first PXE boot only**) kernel parameters to provide a URL to the configuration (also `flatcar.config.url` OR `coreos.config.url` are accepted). The URL can use the `http://`, `https://`, `tftp://`, or `s3://` schemes to specify a remote config or the `oem://` scheme to specify a local config, rooted in `/usr/share/oem`.
```

Comment:
Update `coreos.first_boot=1` to `flatcar.first_boot=1`

Reply:
ah, i didn't see this in the ignition codebase, but I see https://github.com/flatcar-linux/bootengine/blob/45a62e85b7c40e322bd814ce324e5c330e654ce0/dracut/30ignition/ignition-generator#L38